models/_sector/dex/trades/base/_schema.yml

```diff
@@ -0,0 +1,24 @@
+version: 2
+
+models:
+  - name: uniswap_v3_base_base_trades
```

Comment:
this naming standard is very confusing as we have **base** blockchain 😬

Reply:
Yes, do we have any better naming options?

Reply:
I agree it's a bit confusing, but it should remain consistent with the nft structure imo to avoid having too different structures. Probably something like `raw`?

Reply:
`core_` / `root_` / `master_` / `anchor_` / `initial_` / `raw_`

Reply:
Those are good ideas, I will wait for other teammates' decision. I think `raw_` is a good choice.

Reply:
actually i kind of don't like `raw_`. it's just underlying fundamental macro which has some transformations. and raw is raw :)

Reply:
or `fundamental_`?

Reply:
How about staging_ or stg_ ?

Reply:
i had also started typing out `staging_` or `stg_` then took it back. maybe staging is the right approach.
`dex.trades`
`dex.stg_trades`
`dex_<blockchain>.stg_trades`
`uniswap_<version>_<blockchain>.stg_trades`

Reply:
i think staging is the best option from of all the above. I also like just simple `stage_` prefix

Reply:
but.. shouldn't it be postfix instead of prefix?

Reply:
i just pushed with `stg` 😅 but open to `stage` too

Reply:
I like `stg_` - short and sweet

Also, if there's more staging models in one folder, the "main" one can be pushed to the top of the list by adding extra underscore (so `stg__`) - that way it'll stay together with _schema and _sources files.